webapp/urllib3

```diff
@@ -0,0 +1 @@
+/usr/local/bin/../../../Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/urllib3
```

Comment:
Isn't there a standard place where to put these libraries?

Reply:
not sure. I followed the instructions posted above and tried multiple things. 
I'll do the gitignore so that they don't clutter the repo, and they are already deployed in the prod app, but to test individually, everyone will need to add accordingly. Sounds good?

Reply:
Yeah sounds good to me 👍